kloppy/infra/serializers/tracking/epts/serializer.py

```diff
@@ -41,15 +41,27 @@ def _frame_from_row(row: dict, meta_data: EPTSMetaData) -> Frame:
         away_team_player_positions = {}
         for player in meta_data.players:
             if player.team == Team.HOME:
-                home_team_player_positions[player.jersey_no] = Point(
-                    x=row[f'player_home_{player.jersey_no}_x'],
-                    y=row[f'player_home_{player.jersey_no}_y']
-                )
+                if f'player_home_{player.jersey_no}_x' in row:
+                    home_team_player_positions[player.jersey_no] = Point(
+                        x=row[f'player_home_{player.jersey_no}_x'],
+                        y=row[f'player_home_{player.jersey_no}_y']
+                    )
+                else:
```

Comment:
Good one. I would prefer not storing the data at all when info is not there. So just skip the else part. Could you also add a test case with a small file that would break previous implementation?